src/Bridges/Variable/kernel.jl

```diff
@@ -6,21 +6,21 @@ struct KernelBridge{T,M} <: MOI.Bridges.Variable.AbstractBridge
 end
 
 function MOI.Bridges.Variable.bridge_constrained_variable(
-    ::Type{KernelBridge{T}},
+    ::Type{KernelBridge{T,M}},
     model::MOI.ModelLike,
     set::SOS.WeightedSOSCone{M},
 ) where {T,M}
     variables = Vector{MOI.VariableIndex}[]
     constraints = MOI.ConstraintIndex{MOI.VectorOfVariables}[]
-    acc = MA.Zero()
+    acc = SA.AlgebraElement(SparseArrays.sparsevec(SA.key_type(typeof(set.basis))[], MOI.ScalarAffineFunction{T}[], length(set.basis)), SA.algebra(set.basis))
     for (gram_basis, weight) in zip(set.gram_bases, set.weights)
         gram, vars, con = SOS.add_gram_matrix(model, M, gram_basis, T)
         push!(variables, vars)
         push!(constraints, con)
-        acc = MA.add_mul!!(acc, weight, gram)
+        MA.operate!(SA.UnsafeAddMul(*), acc, weight, gram)
```

Comment:
@kalmarek This is the other part. This used to first first convert the gram matrix to a monomial-basis polynomial, now it's directly adding term by term with `unsafe_push!`

Reply:
Actually, I think we know it's going to be dense so we can even use a `Vector` instead of `SparseVector` :thinking:

Reply:
Thats up for basis to decide, see e.g. https://github.com/JuliaAlgebra/StarAlgebras.jl/blob/5bd4ae5cde89e4bfc16eb9f78fe0fe3f353e17bb/src/bases.jl#L39C1-L41C4